lib/management/user.test.ts

```diff
@@ -210,4 +210,341 @@ describe('Management User', () => {
       });
     });
   });
+
+  describe('activate', () => {
+    it('should send the correct request and receive correct response', async () => {
+      const httpResponse = {
+        ok: true,
+        json: () => mockMgmtUserResponse,
+        clone: () => ({
+          json: () => Promise.resolve(mockMgmtUserResponse),
+        }),
+        status: 200,
+      };
```

Comment:
I see this part repeats in every test
maybe we can extract it into a utility function?